tests/zfs-tests/tests/functional/acl/posix/posix_001_pos.ksh

```diff
@@ -54,8 +62,12 @@ if [ "$?" -eq "0" ]; then
 	# Should be able to write to file
 	log_must user_run $ZFS_ACL_STAFF1 "echo 'echo test > /dev/null' > $TESTDIR/file.0"
 
+	# Since $TESTDIR is 777, create a new dir with controlled permissions
+	# for testing create new file NOT allowed.
+	log_must mkdir $TESTDIR/dir.0
```

Comment:
It would make sense to explicitly set the permissions so we're not relying on the umask.